winui/Ribbon/RibbonCompactSizing.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Ribbon Compact Sizing in WinUI Ribbon control | Syncfusion
+description: Learn here all about Ribbon Compact Sizing feature of Syncfusion WinUI Ribbon(SfRibbon) control with custom support and more.
+platform: winui
+control: Ribbon
+documentation: ug
+---
+
+# Compact Sizing in WinUI Ribbon
+
+There are two types of spacing support for WinUI Ribbon control.
+1. [Standard](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-standard-sizing) - to accommodate both touch and pointer input.
+2. [Compact](https://docs.microsoft.com/en-us/windows/apps/design/style/spacing#fluent-compact-sizing) - designed primarily to accommodate pointer input.
+
+The Compact sizing feature in the Ribbon control allows you to display the ribbon in a smaller, more condensed form. This can be useful for scenarios where you need to save space on the user interface or make the ribbon more accessible on smaller screens.
+
+## Applying compact sizing for Ribbon control
+
+Compact sizing is implemented through a unique resource dictionary that can be specified in your application at either the page level or a specific layout. The resource dictionary is available in the WinUI Nuget package.
```

Comment:
Compact sizing is implemented by a unique resource dictionary that can be specified in your application at either the page level or a specific layout. The resource dictionary is available in the WinUI NuGet package.

Reply:
Done the changes.